### DISCUSSION TITLE
VS Code / Code for IBM i - getting started - connecting to system

### DISCUSSION THREAD

**Discussion by wood3875** [codefori/vscode-ibmi#1552] [Q&A] [answered]
Help ?

I'm simply trying to connect with userid/password and VS Code is complaining:  **config.privateKeyPath does not exist at given fs path.**  Tried host name, IP, and FQDN - same result.  I'm NOT using a key pair...  VS Code came from employer software center, and then added Code for IBM i extension, that's it.

Version: 1.82.2 (system setup)
Commit: abd2f3db4bdb28f9e95536dfa84d8479f1eb312d
Date: 2023-09-14T05:55:25.390Z
Electron: 25.8.1
ElectronBuildId: 23779380
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.19045

A co-worker suggested proxy, but I don't need/use that for any other system connection.

**Comment by worksofliam** [comment]
Hey @wood3875 - we are working on a fix for this and we're expecting it to be out today or tomorrow.

**Reply by wood3875** [reply]
Thanks Liam...

**Comment by sebjulliand** [comment, marked as answer]
@wood3875 in the meantime, you can open you user settings as JSON 
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/11096890/95d0f2de-24f7-4119-be8d-176c56d468a3)

And look for any line with `"privateKeyPath":""`. You can safely remove these lines, save and then you'll be able to connect.

**Reply by wood3875** [reply]
That works - Thanks !
